target/linux/ath79/dts/ar9341_tplink_tl-mr3420-v2.dts

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "TP-Link TL-MR3420 v2";
+	compatible = "tplink,tl-mr3420-v2", "qca,ar9341";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "Reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_HIGH>;
```

Comment:
according to machine code in ar71xx (target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr841n-v8.c), reset button is active low. Could you double check which one is correct? (GPIO pin values can be observed from /sys/kernel/debug/gpio)

Reply:
> according to machine code in ar71xx (target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr841n-v8.c), reset button is active low. Could you double check which one is correct? (GPIO pin values can be observed from /sys/kernel/debug/gpio)

I already checked on my hardware, GPIOs are TESTED and WORKING.
This is the correct one